lightning/src/onion_message/messenger.rs

```diff
@@ -417,6 +417,22 @@ pub struct PendingOnionMessage<T: OnionMessageContents> {
 	pub reply_path: Option<BlindedMessagePath>,
 }
 
+/// A placeholder value used for paths within the codebase.
+///
+/// PATHS_PLACEHOLDER` is temporarily used as a default value in situations
```

Comment:
Doesn't look like this is temporary.

Reply:
Removed the global variable in [pr3246.06](https://github.com/shaavan/rust-lightning/commits/pr3246.06)
Thanks for the suggestion!